app/vars.ts

```diff
@@ -1,9 +1,11 @@
+import { protractor } from 'protractor';
+
 import { BrowserSync} from './browser-sync';
 import { ElementFinderSync } from './element-finder-sync';
 import { assertElementDoesNotExist, findElement, findElements, findVisible, findVisibles } from './selection';
 import { getActiveElement } from './utility';
 
-export const browserSync = new BrowserSync((global as any).browser);
```

Comment:
Protractor updates the value of this global as new windows are opened, so we need to make sure browserSync also gets the most up-to-date value.